hypothesis-python/src/hypothesis/internal/floats.py

```diff
@@ -19,7 +19,51 @@
 
 import math
 
-from hypothesis.internal.compat import struct_pack, struct_unpack
+from hypothesis.internal.compat import CAN_PACK_HALF_FLOAT, quiet_raise, \
+    struct_pack, struct_unpack
+
+try:
+    import numpy
+except ImportError:
+    numpy = None
+
+
+STRUCT_FORMATS = {
```

Comment:
Nit: It would be nice to have a comment here explaining what these are (int vs float), as it lacks the context where defined here.